extensions/src/test/java/org/pentaho/platform/web/http/filters/ProxyTrustingFilterTest.java

```diff
@@ -46,6 +50,11 @@
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.times;
+import static org.pentaho.platform.web.http.filters.ProxyTrustingFilter.CSRF_OPERATION_NAME;
```

Comment:
A test is missing that tests the case where AccessDeniedException is thrown.

Reply:
addressed review comments